docs/sce_file_contents.md

```diff
@@ -45,13 +45,17 @@ The following per-cell data columns are included for each cell, calculated using
 | `subsets_mito_percent`  | Percent of all UMI counts assigned to mitochondrial genes                                                                                                                                     |
 | `total`                 | Total UMI count for RNA-seq data and any alternative experiments (i.e., CITE-seq)                                                                                                             |
 
-The following are additional per-cell data columns included only in `filtered` objects.
-These metrics were calculated by using [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells.
+The following are additional per-cell data columns included in both the `filtered` and `processed` objects.
+The included metrics were calculated by [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells.
```

Comment:
My initial edit here was intended to make the statement about metrics less definitive about the source of the data, but I left in the "were"... But then I realize that even with my intended change, that left an unqualified "these", so here I am attempting once again to say what I mean.

```suggestion
The following additional per-cell data columns are included in both the `filtered` and `processed` objects.
These columns include metrics calculated by [`miQC`](https://bioconductor.org/packages/release/bioc/html/miQC.html), a package that jointly models proportion of reads belonging to mitochondrial genes and number of unique genes detected to predict low-quality cells.
```